src/test/java/uk/gov/hmcts/reform/iahearingsapi/domain/entities/hmc/caselinking/caselinking/CaseLinkInfoTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.iahearingsapi.domain.entities.hmc.caselinking.caselinking;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.iahearingsapi.domain.entities.hmc.caselinking.CaseLinkDetails;
+import uk.gov.hmcts.reform.iahearingsapi.domain.entities.hmc.caselinking.CaseLinkInfo;
+
+@ExtendWith(MockitoExtension.class)
+public class CaseLinkInfoTest {
+
+    @Mock
+    private CaseLinkDetails caseLinkDetails;
+
+    @Test
+    void should_have_correct_values() {
```

Comment:
Not sure what the plans are for these entities classes but looking at them now it doesn't seem necessary to have unit tests for them